### PR TITLE
Pinning versions for 2.0.0

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
     <StreamJsonRpcVersion>1.1.92</StreamJsonRpcVersion>
-    <TestSdkVersion>15.3.0-*</TestSdkVersion>
+    <TestSdkVersion>15.3.0</TestSdkVersion>
 
     <!-- Must keep in sync with tooling\Microsoft.VisualStudio.RazorExtension\project.json -->
     <VsShellVersion>15.0.26606</VsShellVersion>
     
-    <XunitVersion>2.3.0-beta2-*</XunitVersion>
+    <XunitVersion>2.3.0-beta2-build3683</XunitVersion>
+    <XunitRunnerVisualStudioVersion>2.3.0-beta2-build1317</XunitRunnerVisualStudioVersion>
```
Comment (pranavkm) [inline build/dependencies.props]: These are two different versions. Had to add these to distinguish between the two packages we reference. The remainder are pretty straighforward.

File: version.props [hunk 2]
```diff
@@ -2,6 +2,5 @@
 <Project>
     <PropertyGroup>
         <VersionPrefix>2.0.0</VersionPrefix>
-        <VersionSuffix>rtm</VersionSuffix>
```
Comment (pranavkm) [inline version.props]: Makes `2.0.0` versioned package.

File: NuGet.config [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />
     <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
```
Comment (smitpatel) [inline NuGet.config]: Do we still need tools in NuGet feed?

Comment (pranavkm) [inline NuGet.config]: The SDK is in the tools feed. We treat it the same as master and do not have a archival policy configured on it.

**Pull request closed without merging**